Pages/FoodCustomer/Food.cshtml.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace FoodKept.Pages
 {
-    [Authorize(Roles ="Customer, Admin, Restaurant")]
```

Comment:
Why "Restaurant" role is deleted? Do we have 2 roles now?

Reply:
Restaurant role is not deleted. It is just made that people who registered as 'Restaurant' could not access the page FoodCustomer/Food, because this page is for customers, not for restaurants